kuma/javascript/src/bcd.js

```diff
@@ -271,45 +271,722 @@ export function activateBCDSignals(
+
+        return signalStepTwoBlock;
+    };
+
+    /**
+     * Builds and returns container for bc signal elements
+     * @returns Signal container block as a `HTMLElement`
+     */
+    const signalStepsBlock = () => {
+        bcSignalBlock = document.createElement('div');
+        bcSignalBlock.className = 'column-container bc-signal-block';
+
+        bcSignalBlock.appendChild(createLeftBlock());
+        bcSignalBlock.appendChild(createFormControlBlock());
+
+        return bcSignalBlock;
+    };
+
+    /**
+     * Builds and returns block which will be shown when user submits report
+     * @returns Complete block as a `HTMLElement`
+     */
+    const signalCompleteBlock = () => {
+        bcSignalCompleteBlock = document.createElement('div');
+        const completeLeftBlock = document.createElement('div');
+        const completeRightBlock = document.createElement('div');
+        const completeImageBlock = document.createElement('div');
+        const navigationButtons = document.createElement('div');
+        const completeRightBlockDescription = document.createElement('div');
+        const completeImageTextBlock = document.createElement('h2');
+        const completeRightTextBlock = document.createElement('h3');
+        const finishButton = document.createElement('button');
+        const closeButtonWrapper = document.createElement('div');
+        const closeButton = document.createElement('button');
+
+        closeButton.className = 'button close-btn';
+        navigationButtons.className = 'navigation-buttons';
+        closeButtonWrapper.className = 'close-button-wrapper';
+        bcSignalCompleteBlock.className =
+            'column-container bc-signal-block complete';
+        completeImageTextBlock.className = 'complete-left-block-header';
+        completeLeftBlock.className = 'column-half left-block';
+        completeRightTextBlock.className = 'right-block-header';
+        completeRightBlock.className = 'column-half right-block';
+        completeImageBlock.className = 'complete-left-block-image';
+        completeRightBlockDescription.className = 'right-block-main-text';
+        completeImageBlock.style.display = 'block';
+        finishButton.className = 'button neutral main-btn';
+        finishButton.innerText = gettext('Finish');
+
+        finishButton.addEventListener('click', finishReport);
+        completeImageTextBlock.innerText = gettext('Thank you!');
+        completeRightTextBlock.innerHTML = `<span>${gettext(
+            'Report sent'
+        )}</span>`;
+        completeRightBlockDescription.innerHTML = `
+            <b>${gettext('What happens next?')}</b><br />
+            <p>
+                ${gettext(
+                    'Our team will review your report. Once we verify the information you have supplied we will update this browser compatability table accordingly.'
+                )}
+            </p>
+            <b>${gettext('Can I keep track of my report?')}</b><br />
+            </p>
```

Comment:
```suggestion
            <p>
```

Reply:
By the way, this would have been avoided if this was JSX instead.

Reply:
Sounds good. Will add this update.